connect/mirror/src/main/java/org/apache/kafka/connect/mirror/DefaultGroupFilter.java

```diff
@@ -73,16 +72,11 @@ static class GroupFilterConfig extends AbstractConfig {
                     Type.LIST,
                     GROUPS_EXCLUDE_DEFAULT,
                     Importance.HIGH,
-                    GROUPS_EXCLUDE_DOC)
-            .define(GROUPS_EXCLUDE_CONFIG_ALIAS,
-                    Type.LIST,
-                    null,
-                    Importance.HIGH,
-                    "Deprecated. Use " + GROUPS_EXCLUDE_CONFIG + " instead.");
+                    GROUPS_EXCLUDE_DOC);
 
         GroupFilterConfig(Map<String, ?> props) {
             super(DEF, ConfigUtils.translateDeprecatedConfigs(props, new String[][]{
```

Comment:
we don't need to call `translateDeprecatedConfigs` now, since the deprecated configs are removed